express-test/docker-compose.yml

```
@@ -0,0 +1,15 @@
+version: '2'
```

If no compatibility is needed I suggest to just bump it to latest Docker Compose specification, i.e. version `3`:

```suggestion
version: '3'
```